src/runtime-rs/crates/resource/src/manager.rs

```diff
@@ -105,6 +105,11 @@ impl ResourceManager {
         let inner = self.inner.read().await;
         inner.delete_cgroups().await
     }
+
+    pub async fn clean_up_hostpath(&self) -> Result<()> {
```

Comment:
How about changing `clean_up` to `cleanup`, there are already many `cleanup`s.

Reply:
If we may need cleanup other resources in the future, I'd like the suggestion to add a method named `cleanup` which may do many cleanups including host path by calling private methods like:

```rust
pub cleanup(){
    self.cleanup_hostpath;
    self.cleanup_vlume;
    self.cleanup_network;
    ... ...
}
```

Just like https://github.com/kata-containers/kata-containers/pull/6080/files#diff-c62b06f33ee88853baa89dbb1c7b84e5da4d75e18172938d29475a81b0289f6bR262 what has done.

Reply:
fixed